lib/rails/observers/active_model/observing.rb

```diff
@@ -14,8 +14,13 @@ module Observing
 
     included do
       extend ActiveSupport::DescendantsTracker
+      def self.inherited(subclass)
+        super
+        ActiveRecord::Base.instantiate_observers
```

Comment:
@mvastola , @rafaelfranca : Just took a look at this and and the Travis tests fail because in the context of the active_resource_observer_test, ActiveRecord::Base is not defined at all and therefore not found. 
I'm not clear whether the gem can be used without activerecord or not. I'm also not sure if we can use Object.const_defined?('ActiveRecord::Base') to verify existence of ActiveRecord::Base...

Either way, this fixes all tests, you guys might have a more elegant solution to it...:

```
ActiveRecord::Base.instantiate_observers if Object.const_defined?('ActiveRecord::Base')
```